src/test/java/com/ryanharter/auto/value/parcel/AutoValueParcelExtensionTest.java

```diff
@@ -159,12 +159,18 @@
         "import java.lang.String;\n" +
         "\n" +
         "final class AutoValue_Test extends $AutoValue_Test {\n" +
-        "  private static final ClassLoader CL = AutoValue_Test.class.getClassLoader();\n" +
+        "  static final ClassLoader CL = AutoValue_Test.class.getClassLoader();\n" +
```

Comment:
This is made packaged scope to avoid a new synthetic accessor method being generated since it's now referenced from inside `CREATOR`.

In a follow-up I'm going to change this field to be conditionally generated only when actually needed, and as a local inside `createFromParcel` since there no reason to be wasting memory holding the reference all the time.